testapi.pm

```diff
@@ -2280,6 +2282,8 @@ sub upload_asset {
     bmwqemu::log_call(file => $file, public => $public, nocheck => $nocheck);
     my $cmd = "curl --form upload=\@$file ";
     $cmd .= "--form target=assets_public " if $public;
+    # Redirect the output so the progress meter is displayed 
+    $cmd .= "-o /dev/$serialdev " if get_var('UPLOAD_METER');
```

Comment:
this is duplication. Even if it's just two lines better extract into a function that alters the variable `$cmd`